examples/example-gateway/endpoints/baz/silly_noop.json

```diff
@@ -0,0 +1,14 @@
+{
+	"endpointType": "http",
+	"endpointId": "baz",
+	"handleId": "sillyNoop",
+	"thriftFile": "endpoints/baz/baz.thrift",
+	"thriftFileSha": "{{placeholder}}",
+	"thriftMethodName": "SimpleService::SillyNoop",
+    "workflowType": "tchannelClient",
```

Comment:
you keep mixing spaces vs tabs. 

Can you evaluated [eclint](https://github.com/jedmao/eclint) and add it to zanzibar for .json and .tmpl files ?